packages/components/clipboard/src/components/osds-clipboard/core/controller.ts

```diff
@@ -1,8 +1,50 @@
+import type { OsdsClipboard } from '../osds-clipboard';
 import { writeOnClipboard } from '@ovhcloud/ods-common-core';
 
 class OdsClipboardController {
+  private component: OsdsClipboard;
+
+  constructor(component: OsdsClipboard) {
+    this.component = component
+  }
+
   async handlerClick(value: string): Promise<void> {
-    await writeOnClipboard(value);
+    const successMessage = this.component.el.querySelector('[slot=success-message]')?.innerHTML;
+    const errorMessage = this.component.el.querySelector('[slot=error-message]')?.innerHTML;
+
+    try {
+      writeOnClipboard(value);
```

Comment:
either use `await` here or use `.then` / `.catch` but here the promise is just iggnored

Reply:
"await" has been added